Apps/Sandcastle/gallery/development/Picking Sublayer Regions.html

```diff
@@ -0,0 +1,174 @@
+        waitSeconds : 60
+    });
+    </script>
+    <style type="text/css">
+        @import url(../../templates/bucket.css);
+    </style>
+</head>
+<body class="sandcastle-loading" data-sandcastle-bucket="bucket-requirejs.html">
+<div id="cesiumContainer" class="fullSize"></div>
+<div id="loadingOverlay"><h1>Loading...</h1></div>
+<div id="toolbar"></div>
+<script id="cesium_sandcastle_script">
+function startup(Cesium) {
+"use strict";
+//Sandcastle_Begin
+
+var viewer = new Cesium.Viewer('cesiumContainer', { infoBox : false });
+var entities = viewer.entities;
+var scene = viewer.scene;
+var handler = new Cesium.ScreenSpaceEventHandler(scene.canvas);
+
+var pickedObject;
+var stack = {
+    boxes: 10,
+    location: {lon: -70.625353, lat: 41.938152, alt: 0},
+    exterior: {height: 129, length: 15, width: 15},
+    interior: {height: 11.99, length: 14.9, width: 14.9, spacing: 1},
+    entity: entities.add({})
+};
+
+//offset heights to work with bottom height instead of center height of Geometry
+var offsetExteriorHeight = stack.exterior.height / 2;
+var offsetInteriorHeight = stack.interior.height / 2;
+
+//Visible Building Model
+var exterior = entities.add({
+    parent : stack.entity,
+    id: 'stack-exterior',
+    position: Cesium.Cartesian3.fromDegrees(
+            stack.location.lon,
+            stack.location.lat,
+            stack.location.alt + offsetExteriorHeight),
+    box: {
+        dimensions : new Cesium.Cartesian3(
+                stack.exterior.length,
+                stack.exterior.width,
+                stack.exterior.height),
+        material : Cesium.Color.DARKGREEN,
+        fill: true,
+        outline : false,
+        outlineColor : Cesium.Color.BLACK
+    },
+    label: {
+        text: 'Cesium Tower',
+        show: false
+    }
+});
+
+//collection for hittable entity testing
+var hittable = new Cesium.EntityCollection();
+
+//Non-visible "pick-able" Objects
+var interior = new Cesium.Entity({
+    parent: stack.entity,
+    id: 'stack-blocks',
+    show: true
+});
+
+for (var box = 0; box < stack.boxes; box++) {
+    var heightInStack = (box * stack.interior.height) + (box * stack.interior.spacing);
+    entities.add(
+            hittable.add({
+                parent : interior,
+                id: 'stack-blocks-' + box,
+                name: 'Contained Box ' + (box+1),
+                label: {
+                    text: 'Contained Box: ' + (box+1),
+                    show: false,
+                    eyeOffset: new Cesium.Cartesian3(25.0, 0.0, -25.0)
+                },
+                position : Cesium.Cartesian3.fromDegrees(
+                        stack.location.lon,
+                        stack.location.lat,
+                        heightInStack + offsetInteriorHeight),
+                box : {
+                    dimensions : new Cesium.Cartesian3(
+                            stack.interior.length,
+                            stack.interior.width,
```

Comment:
If you add `+ 0.2` to `.length` and `.width` here, you can get the visible box to stick out a bit from the main box.  This way if the user toggles the interior outlines on, they can still see them even with a solid exterior.